pyproject.toml

```diff
@@ -34,21 +47,30 @@ Source = "https://github.com/lsst-sqre/nublado"
 requires = ["setuptools>=61", "setuptools_scm[toml]>=6.2"]
 build-backend = "setuptools.build_meta"
 
+[project.scripts]
+rsp_purger = "nublado.purger.cli:main_with_sentry"
+
 [dependency-groups]
 dev = [
+    "anys",
+    "asgi-lifespan",
+    "coverage[toml]",
+    "httpx-sse",
     "pyfakefs",
     "pytest",
     "pytest-asyncio",
     "pytest-cov",
     "pytest-sugar",
+    "pytest-timeout",
+    "respx",
 ]
 docs = [
     "documenteer[guide]>=2.3",
     "scriv[toml]>=1.5",
     "sphinx-diagrams",
     # Required to generate API documentation.
-    "controller",
     "respx",
+    "nublado",
```

Comment:
You can delete this line. The module is always installed in the virtualenv and doesn't need to be explicitly given.